lua/lvim/plugins.lua

```diff
@@ -52,6 +55,9 @@ return {
   {
     "rcarriga/nvim-notify",
     commit = commit.nvim_notify,
+    config = function()
+      require("lvim.core.notify").setup()
+    end,
```

Comment:
Let's make this a separate PR.

Reply:
This is necessary to prevent install from erroring out because no default config is provided _at all_. The default animation requires a background to be set for Normal.